servers/quarkus-server/pom.xml

```diff
@@ -165,6 +165,14 @@
       <groupId>io.quarkus</groupId>
       <artifactId>quarkus-container-image-jib</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-kubernetes</artifactId>
+    </dependency>
+    <dependency>
```

Comment:
why would minikube be a dependency?

Reply:
it generate Kubernetes manifests (minikube.yaml and minikube.json) that are tailored to Minikube during the build

Reply:
So w/o this we can't deploy to minikube?  do they get packaged into the jar file or docker image or just left in `target`?

Reply:
correct, we can't deploy the generated kubernetes manifest with minikube, since that plugin adds some stuff necessary for it to work with minikube (which lands under `target/kubernetes/minikube.yml`), but it doesn't add anything to the final `jar` and everything stays under `target`